### PR TITLE
docs: Issue Metrics and SLA Enhancements

### PR DESCRIPTION
**Pull request by ruchamahabal**
Documentation for: https://github.com/frappe/erpnext/pull/21617

### CODE REVIEW

File: erpnext_documentation/www/docs/user/manual/en/support/issue.md [hunk 1]
```diff
@@ -32,10 +32,13 @@ replied, its status becomes "Replied".
     * Open: The Issue has been created and yet to be replied to.
     * Replied: A reply has been sent to the Issue.
     * Hold: The Issue is on Hold due to some reason.
+    * Resolved: When the users are reasonably sure that they have provided the customer with a solution to their problem but have not acknowledgement about the resolution from the customer.
     * Closed: The Customer got a resolution and the Issue was closed.
 
     If the sender replies to the thread, the status becomes "Open" again. User can "Close" the Issue manually by clicking on the **Close** button on the top right.
 
+> Note: If SLA has been set up, then the fulfillment status of the SLA will be updated on both, the **Closed** as well as the **Resolved** status. The key difference between these two statuses is to indicate the case where the user solving the issue is sure about the resolution and the customer has not yet acknowledged that the issue has been resolved (Resolved) and the case where an acknowledgement is received (Closed).
```
Comment (akshitakl) [inline erpnext_documentation/www/docs/user/manual/en/support/issue.md:40]: If SLA has been set up, then the fulfillment status of the SLA will be updated on both **Closed** as well as **Resolved** status. 

The second line can be removed as you are already adding that above.

File: erpnext_documentation/www/docs/user/manual/en/support/issue.md [hunk 2]
```diff
@@ -32,10 +32,13 @@ replied, its status becomes "Replied".
     * Open: The Issue has been created and yet to be replied to.
     * Replied: A reply has been sent to the Issue.
     * Hold: The Issue is on Hold due to some reason.
+    * Resolved: When the users are reasonably sure that they have provided the customer with a solution to their problem but have not acknowledgement about the resolution from the customer.
     * Closed: The Customer got a resolution and the Issue was closed.
```
Comment (akshitakl) [inline erpnext_documentation/www/docs/user/manual/en/support/issue.md:36]: The Customer got a satisfactory resolution that he has indicated with an acknowledgment and the Issue was closed.

File: erpnext_documentation/www/docs/user/manual/en/support/issue.md [hunk 3]
```diff
@@ -58,10 +61,14 @@ User can select the [Service Level Agreement](/docs/user/manual/en/support/servi
 <img class="screenshot" alt="Issue" src="{{docs_base_url}}/assets/img/support/iss.gif">
 
 ### 3.3 Response
-**Mins to First Response**: Time in minutes from when the Issue was created to when the first reply was sent.
+* **Mins to First Response**: Time in minutes from when the Issue was created to when the first reply was sent.
 
 * **First Responded on**: When a Support team member first replies to the issue, first response date and time will be updated.
 
+* **Average Response Time**: The average time taken to respond to the Customer. This is calculated as the average of all time spans between Received and Sent Communications. This field will be updated on every reply to the customer.
```
Comment (akshitakl) [inline erpnext_documentation/www/docs/user/manual/en/support/issue.md:68]: This is the average time between all sent and received communications

File: erpnext_documentation/www/docs/user/manual/en/support/issue.md [hunk 4]
```diff
@@ -76,6 +83,14 @@ User can filter the issues based on these fields linked to the Issue:
 * **Opening Time**: When the issue is created or logged the exact time will automatically be posted.
 * **Resolution Date**: When the user resolved the issue, Date and Time will get updated in this field.
 * **Resolution Details**: User can enter the details of the issue, once it is resolved. This is a text field. Also, user can upload the image, enter or create a table.
+* **Resolution Time**: Total time taken to close the ticket (from Issue creation to closing).
+* **User Resolution Time**: Many times a user has to wait for a customer's reply in order to solve some Issue. While measuring User's productivity this wait time shouldn't be taken into consideration. Hence, user resolution time is the total time taken by a user to close the ticket calculated as:
```
Comment (akshitakl) [inline erpnext_documentation/www/docs/user/manual/en/support/issue.md:87]: Hence, user resolution time is the total time taken by a user to close the ticket and can be calculated as:

File: erpnext_documentation/www/docs/user/manual/en/support/issue.md [hunk 5]
```diff
@@ -76,6 +83,14 @@ User can filter the issues based on these fields linked to the Issue:
...
+
+    _Resolution time - Total Time for which the user had to wait for a customer's reply_
+
+The Resolution Time and User Resolution Time metrics are set on "Close" and reset when the Issue reopens or splits.
```
Comment (akshitakl) [inline erpnext_documentation/www/docs/user/manual/en/support/issue.md:91]: The Resolution Time and User Resolution Time metrics are set on "Close". These metrics reset automatically when the Issue reopens or splits.

File: erpnext_documentation/www/docs/user/manual/en/support/service-level-agreement.md [hunk 6]
```diff
@@ -9,21 +9,30 @@ To access the Service Level Agreement list, go to:
...
 
 * Enable **Track Service Level Agreement** in Support Settings
 
     <img class="screenshot" alt="Service Level Agreement" src="{{docs_base_url}}/assets/img/support/sla-setting.png">
 
 ## 2. How to Create a Service Level Agreement
 1. Go to the Service Level Agreement list, click on New.
-1. Select the Service Level. This will fetch the Employee Group the SLA applies to, the default priority, and the Holiday list the SLA is excluded from.
+1. Enter a name for the Service Level.
+1. Select an Employee group, which will handle a particular Service Level.
```
Comment (akshitakl) [inline erpnext_documentation/www/docs/user/manual/en/support/service-level-agreement.md:24]: Select an Employee group, that will handle a particular Service Level.

File: erpnext_documentation/www/docs/user/manual/en/support/issue.md (hunk 3 above)
Comment (ruchamahabal) [reply erpnext_documentation/www/docs/user/manual/en/support/issue.md:68]: Actually it is specifically "Response Time" so it is better if the definition is in detail

Comment (akshitakl) [reply erpnext_documentation/www/docs/user/manual/en/support/issue.md:68]: The average time taken to respond to the Customer. This is calculated by taking the average of all time spans between Received and Sent Communications. This field will be updated on every reply to the customer.

**Comment by akshitakl** [comment]
Spell check fails: SLA

**Pull request merged**